app/services/idv/aamva_state_maintenance_window.rb

```diff
@@ -0,0 +1,165 @@
+# frozen_string_literal: true
+
+module Idv
+  class AamvaStateMaintenanceWindow
+    # _All_ AAMVA maintenance windows are expressed in 'ET' (LG-14028)
+    TZ = 'America/New_York'
+
+    MAINTENANCE_WINDOWS = {
+      'CA' => [
+        # Daily, 4:00 - 5:30 am. ET.
+        { cron: '0 4 * * *', duration: 90 },
```

Comment:
WDYT about renaming `duration:` to something with units?
```suggestion
        { cron: '0 4 * * *', duration_minutes: 90 },
```

Reply:
That seems a lot more expressive. I like it.